client-side/src/router/router.jsx

```diff
@@ -16,13 +16,13 @@ export  const router = createBrowserRouter([
                 path: '/home',
                 element: <h1>home</h1>
             },
-            {
-                path: '/profiles',
-                element:<ProfileList/>
-            },
             {
                 path: '/login',
                 element: <Login/>
+            },
+            {
+                path: '/reports',
+                element: <ManagerGoogleDrive/>
```

Comment:
this is not the reports UI, this should be only a button inside the reports page